wagtail/wagtailadmin/userbar.py

```diff
@@ -24,7 +24,7 @@ def render(self, request):
             return ""
 
         # Don't render if user doesn't have ability to add siblings
```

Comment:
Comment should be changed here "# Don't render if user doesn't have ability to add **children**"